scripts/dispatcher.py

```diff
@@ -0,0 +1,33 @@
+"""
+usage: teuthology-dispatcher --help
+       teuthology-dispatcher --supervisor [-v] --bin-path BIN_PATH --job-config COFNFIG --archive-dir ARC_DIR
+       teuthology-dispatcher [-v] --archive-dir ARC_DIR --log-dir LOG_DIR --tube TUBE
+
+Start a dispatcher for the specified tube. Grab jobs from a beanstalk
+queue and run the teuthology tests they describe as subprocesses. The
+subprocess invoked is a teuthology-dispatcher command run in supervisor
+mode.
+
+Supervisor mode: Supervise the job run described by its config. Reimage
+target machines and invoke teuthology command. Unlock the target machines
+at the end of the run.
+
+standard arguments:
+  -h, --help                     show this help message and exit
+  -v, --verbose                  be more verbose
+  -t, --tube TUBE                which beanstalk tube to read jobs from
+  -l, --log-dir LOG_DIR          path in which to store logs
+  --archive-dir ARC_DIR          path to archive results in
+  --supervisor                   run dispactcher in job supervisor mode
+  --bin-path BIN_PATH            teuthology bin path
+  --job-config CONFIG                 file descriptor of job's config file
+"""
```

Comment:
When you run teuthoogy-worker:
```
teuthology-worker --help
usage: teuthology-worker [-h] [-v] --archive-dir DIR -l LOG_DIR -t TUBE

Grab jobs from a beanstalk queue and run the teuthology tests they describe. One job is run at a time.

optional arguments:
  -h, --help            show this help message and exit
  -v, --verbose         be more verbose
  --archive-dir DIR     path under which to archive results
  -l LOG_DIR, --log-dir LOG_DIR
                        path in which to store logs
  -t TUBE, --tube TUBE  which beanstalk tube to read jobs from
```
or teuthology:
```
teuthology --help
usage: teuthology --help
       teuthology --version
       teuthology [options] [--] <config>...

Run ceph integration tests

positional arguments:
  <config> one or more config files to read

optional arguments:
  -h, --help                     show this help message and exit
  -v, --verbose                  be more verbose
  --version                      the current installed version of teuthology
  -a DIR, --archive DIR          path to archive results in
  --description DESCRIPTION      job description
  --owner OWNER                  job owner
  --lock                         lock machines for the duration of the run
  --machine-type MACHINE_TYPE    Type of machine to lock/run tests on.
  --os-type OS_TYPE              Distro/OS of machine to run test on.
  --os-version OS_VERSION        Distro/OS version of machine to run test on.
  --block                        block until locking machines succeeds (use with --lock)
  --name NAME                    name for this teuthology run
  --suite-path SUITE_PATH        Location of ceph-qa-suite on disk. If not specified,
                                 it will be fetched
```

You may see there is archive dir, like:
```
 -a DIR, --archive DIR
```
Could you please align teuthology-dispatcher to use the same style, i.e. use `DIR` instead of `ARC_DIR` and, it ould be great to have short option for it.

Reply:
done.

Reply:
Done.